File: C++/First Steps in Coding - Basics of C++/Nested Loops/Travelling.cpp
#include <iostream>
#include <string>
using namespace std;
int main() {
	double budget = 0;
	while (true) {
		string destination;
		getline(cin>>ws, destination);
		if (destination != "End") {
			double minBudget;
			cin >> minBudget;
			while (budget < minBudget) {
				double sum;
				cin >> sum;
				budget += sum;
			}
			cout << "Going to " << destination << "!" << endl;
			budget = 0;
		}
		else {
			break;
		}
	}
}

File: Python/Basics/Nested Loops/Lab/Travelling.py
savings = 0

while True:
    destination = input()

    if destination != "End":
        minBudget = float(input())

        while True:
            sum = float(input())
            savings += sum

            if savings >= minBudget:
                print(f"Going to {destination}!")
                break

        savings = 0
    else:
        break